docs/user-guide/managing-deployments.md

```diff
@@ -439,6 +441,27 @@ $ rm $TMP
 
 The system ensures that you don’t clobber changes made by other users or components by confirming that the `resourceVersion` doesn’t differ from the version you edited. If you want to update regardless of other changes, remove the `resourceVersion` field when you edit the resource. However, if you do this, don’t use your original configuration file as the source since additional fields most likely were set in the live state.
 
+## Using configuration files
+
+A more disciplined alternative to patch and edit is `kubectl apply`. 
+
+With apply, you can keep a set of configuration files in source control, where they can be maintained and versioned along with the code for the resources they configure. Then, when you're ready to push configuration changes to the cluster, you can run `kubectl apply`.
+
+This command will compare the version of the configuration that you're pushing with the previous version and apply the changes you've made, without overwriting any automated changes to properties you haven't specified.
+
+```console
+$ kubectl apply -f ./nginx-rc.yaml
+replicationcontrollers/my-nginx-v4
```

Comment:
Is this the actual output of apply? This is the old output convention rather than the new one.